src/posthog-featureflags.ts

```diff
@@ -391,20 +391,18 @@ export class PostHogFeatureFlags {
             data.disable_flags = true
         }
 
-        // NB: flags v2 requires remote config to be enabled as well, since the idea is that we will skip calling /decide altogether
-        // (which remote config does) and just use the /flags endpoint.  May revisit this if we need to support flags v2 without remote config
-        // (e.g. we could call `/decide` with flags disabled for the data otherwise returned by remote config, and then still call
-        // `/flags/` to get the flag evaluation data).
-        const eligibleForFlagsV2 =
+        // Flags v2 requires remote config to be enabled to skip /flags&config=true and use /flags directly
+        const useRemoteConfigWithFlags =
             this._instance.config.__preview_flags_v2 && this._instance.config.__preview_remote_config
 
-        const baseUrl = eligibleForFlagsV2 ? '/flags/?v=2' : '/decide/?v=4'
+        // if we're not using RemoteConfig (which loads the config info from the JS snippet itself), then get all the data from remote config on the server side
+        const baseUrl = useRemoteConfigWithFlags ? '/flags/?v=2' : '/flags/?v=2&config=true'
```

Comment:
@dmarticus @lucasheriques we need to make sure that surveys work in both cases, with remote config enabled or not

Reply:
seems to work on my testing, I did the following scenarios:

- turning on surveys yields "surveys": true
- survey flag targeting works

What was the situation that wasn't working in the past?

Reply:
There wasn't a problem before.
I just wanted to ensure you test both scenarios, as flipping the remote config flag would use a different endpoint (adding a new query param, actually) that responds with a more complete configuration. If surveys still work in both scenarios, that's all good.

Reply:
Yeah, surveys still work in both cases :) `/flags&config=true` is basically `/decide` now.